src/components/DigitalCredentials/dialogs/ConfirmCredentialsTermsofUseDialog.vue

```diff
@@ -53,53 +42,44 @@
 
 <script lang="ts">
 import { Component, Vue, Prop, Emit } from 'vue-property-decorator'
+import CredentialsTermsOfUse from '@/components/DigitalCredentials/CredentialsTermsOfUse.vue'
 
-  @Component({})
+@Component({
+  components: {
+    CredentialsTermsOfUse
+  }
+})
 export default class ConfirmCredentialsTermsOfUseDialog extends Vue {
-    /** Prop to display the dialog. */
-    @Prop({ default: false }) readonly dialog!: boolean
+  /** Prop to display the dialog. */
+  @Prop({ default: false }) readonly dialog!: boolean
 
-    /** Prop to provide attachment selector. */
-    @Prop({ default: '' }) readonly attach!: string
+  /** Prop to provide attachment selector. */
+  @Prop({ default: '' }) readonly attach!: string
 
-    // Pass click events to parent.
-    @Emit() close (): void {
-      return undefined
-    }
-    @Emit() proceed (): void {
-      return undefined
-    }
+  // Pass click events to parent.
+  @Emit() close (): void {
+    return undefined
+  }
+  @Emit() proceed (): void {
+    return undefined
+  }
 }
 </script>
 
-  <style lang="scss" scoped>
-  @import '@/assets/styles/theme.scss';
-
-  .close-btn {
-    position: absolute;
-    right: $px-32;
-  }
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
 
-  .action-btn {
-    min-height: 44px;
-  }
-
-  .warning-title {
-    font-size: $px-24;
-    font-weight: bold;
-    color: $gray9;
-    line-height: $px-34;
-  }
+.close-btn {
+  position: absolute;
+  right: $px-32;
+}
 
-  .warning-text {
-    font-weight: normal;
-    font-size: $px-16;
-    color: $gray7;
-    line-height: $px-24;
-  }
+.action-btn {
+  min-height: 44px;
+}
 
-  // Vuetify overrides
-  :deep(.v-dialog .v-card .v-card__text) {
-    padding-top: 0 !important;
-  }
-  </style>
+// Vuetify overrides
+:deep(.v-dialog .v-card .v-card__text) {
+  padding-top: 0 !important;
+}
+</style>
```

Comment:
The text colour is fine ($gray7) but the titles should be $gray9.

Reply:
Also, should probably have some top and bottom whitespace.

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/9b1be05f-fc80-4350-bfbd-31a227c1a958)